src/main/resources/jenkins/plugins/logstash/LogstashInstallation/global.jelly

```diff
@@ -1,33 +1,3 @@
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <f:section title="Logstash Plugin" name="logstash">
-    <f:entry title="${%Indexer type}" field="type">
-      <f:enum value="${descriptor.type}">${it.name()}</f:enum>
-    </f:entry>
-    <f:entry title="${%Host name}" field="host">
-      <f:textbox value="${descriptor.host}"
-        checkUrl="'${resURL}/descriptorByName/LogstashInstallation/checkHost?value='+escape(this.value)" />
-    </f:entry>
-    <f:entry title="${%Port}" field="port">
-      <f:textbox value="${descriptor.port}" default="6379"
-        checkUrl="'${resURL}/descriptorByName/LogstashInstallation/checkInteger?value='+escape(this.value)" />
-    </f:entry>
-    <f:entry title="${%Username}" field="username">
-      <f:textbox value="${descriptor.username}" />
-    </f:entry>
-    <f:entry title="${%Password}" field="password">
-      <f:password value="${descriptor.password}" />
-    </f:entry>
-    <f:entry title="${%Key}" field="key">
-      <f:textbox value="${descriptor.key}" default="logstash"
-        checkUrl="'${resURL}/descriptorByName/LogstashInstallation/checkString?value='+escape(this.value)" />
-    </f:entry>
-    <f:advanced>
-      <f:entry title="${%Syslog format}" field="syslogFormat">
-        <f:enum value="${descriptor.syslogFormat}">${it.name()}</f:enum>
-      </f:entry>
-      <f:entry title="${%Syslog protocol}" field="syslogProtocol">
-        <f:enum value="${descriptor.syslogProtocol}">${it.name()}</f:enum>
-      </f:entry>
-    </f:advanced>
-  </f:section>
+<!-- We need to keep this file in order to prevent that in the UI we get the option to configure a Logstash Installation -->
```

Comment:
How about leaving some explanatory text here, e.g. "Configuration moved to Global Configuration"? Bonus points for a link :)

Reply:
I think it should be enough to document this on the wiki page. 
btw I suggest to upgrade the version to 2.0. And maybe we find a way to show that plugin changed in a way that when you go back to previous version you have to manually adjust configuration (this is possible somehow).

Reply:
I know they should check the wiki page, but this makes for less frustration.

Can we leave a short message here with a link to the wiki page then?

Reply:
will add a link to global configuration page